### PR TITLE
[DOCS-7396] Governance Services 23.1

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

**Review by anxumalo — CHANGES_REQUESTED** [review]
Looking good - see inline comments

File: governance-services/7.4/upgrade/index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+To upgrade Governance Services you need to make sure you're running the correct version of Alfresco Content Services.
+
+Check the [Supported Platforms]({% link content-services/latest/support/index.md %}) and 
+the [Alfresco Content Services upgrade paths]({% link content-services/latest/upgrade/index.md %}).
+
+When your Alfresco Content Services installation is upgraded, you can apply the new AMP files for Governance Services.
+
+1. Ensure your current production environment is running a version that is supported for upgrading.
+
+2. Download Alfresco Content Services 7.4 or above and the Governance Services Distribution ZIP from [Hyland Community](https://community.hyland.com/){:target="_blank"}.
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply governance-services/7.4/upgrade/index.md:14]: Removed.

File: governance-services/latest/upgrade/index.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ When your Alfresco Content Services installation is upgraded, you can apply the
 
 1. Ensure your current production environment is running a version that is supported for upgrading.
 
-2. Download Alfresco Content Services 7.4 or above and the Governance Services Distribution ZIP from [Hyland Community](https://community.hyland.com/){:target="_blank"}.
+2. Download Alfresco Content Services 23.1 or above and the Governance Services Distribution ZIP from [Hyland Community](https://community.hyland.com/){:target="_blank"}.
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply governance-services/latest/upgrade/index.md:14]: removed.

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**